packages/share_plus/share_plus/README.md

```diff
@@ -49,16 +51,18 @@ sharing to email.
 Share.share('check out my website https://example.com', subject: 'Look what I made!');
 ```
 
-If you are interested in the action your user performed with the share sheet, you can instead use the `shareWithResult` method.
+If you are interested in the action your user performed with the share sheet, the `share` method returns a status object.
```

Comment:
```suggestion
`share()` returns `status` object that allows to check the result of user action in the share sheet.
```

Not sure if it would be better, but felt to me so.